docker-compose.yml

```diff
@@ -1,14 +1,50 @@
-version: '3'
-
 services:
+  postgres_db:
+    image: postgres:16.4-bullseye
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+    volumes:
+      - postgres:/var/lib/postgresql/data
+    networks:
+      - internal_network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 30s
+      timeout: 60s
+      retries: 5
+    env_file:
+      - ./.env
+    restart: always
   backend:
-    image: e889ceef0235d3890a0b9fc582a0a7f8a7b63eeec1399efda7e91bd04534094e
+    build:
+      context: .
+      dockerfile: backend/Dockerfile
+    command: python3 manage.py runserver
```

Comment:
`python3 manage.py runserver 0.0.0.0:8000`

without this idts the frontend can communicate